source/_remoteClient/localMachine.py

```diff
@@ -251,9 +282,55 @@ def sendSAS(self) -> None:
 
 		:note: SendSAS requires UI Access. If this fails, a warning is displayed.
 		"""
-		if hasUiAccess():
-			ctypes.windll.sas.SendSAS(0)
+		if self._canSendSAS():
+			ctypes.windll.sas.SendSAS(not isRunningOnSecureDesktop())
```

Comment:
I'd avoid chaining methods here.
```suggestion
			asUser = not isRunningOnSecureDesktop()
			ctypes.windll.sas.SendSAS(asUser)
```

Reply:
In the current code, one line contains two function calls. For debugging purposes, if either the inner or the outer call raises an error, at first sight it might not be obvious whether the inner or outer call caused the error. That said, This should certainly not be a problem for the experienced reader. Feel free to leave it as is.